src/components/PostsList/PostsList.tsx

```diff
@@ -1,37 +1,83 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './PostsList.scss';
+import { getAllPosts, getPosts } from '../../api/posts';
+import { Post } from '../../react-app-env';
```

Comment:
Add wrong imports position
import React, { useState, useEffect } from 'react';
import { getAllPosts, getPosts } from '../../api/posts';
import { Post } from '../../react-app-env';
import './PostsList.scss';
and add alias pathes